tools/testbench/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ add_executable(testbench
 	trace.c
 )
 
+add_definitions(-DBUILD_HOST=1)
```

Comment:
`BUILD_HOST` seems a bit generic for me. I guess you need it to differentiate testbench vs fuzzer build. Can you make it `BUILD_TESTBENCH` and please use `target_compile_definitions(testbench PRIVATE -D...)`, because add_definitions is going to add it to all targets in this folder and in *possible* future subfolders.

Reply:
@jajanusz fixed now. thanks!